documentation/src/main/asciidoc/contributing/basics.adoc

```diff
@@ -126,7 +126,7 @@ image::images/idea-12-git.png[]
 
 ====== Compiler settings
 
-From Infinispan 5.0 onwards, annotation processing is used to allow log messages to be internationalized.
+From {brandname} 5.0 onwards, annotation processing is used to allow log messages to be internationalized.
```

Comment:
I'm not sure what we can do about this, but `JDG 5.0 != Infinispan 5.0`. This occurs several times. I assume some kind of map from Infinispan > JDG versions would be beyond the filtering capabilities?

Reply:
Yeah I totally agree. We will probably have a ton of placed like that. But at least we can start working on it. Big thumbs up!

Reply:
@slaskawi -1, this would be very confusing. I would rather keep references to upstream version non-branded.
@ryanemerson Simple workaround would be `{version50}` -> `Infinispan 5.0`/`JDG 6.0`

Reply:
I have now done the following:
* in the user guide I have removed all references to "since xxx X.Y" which didn't provide any useful migration/upgrading information.
* where migration/upgrading information is needed, I defer to the upgrading guide for details
* I have hardcoded "Infinispan X.Y" versions where neither of the above applied